assets/constants.js

```diff
@@ -10,6 +10,12 @@
 var constants = {
   // Whether to allow custom event reporting to Google Analytics.
   "CAN_SEND_ANALYTICS_EVENTS": false,
+  /* Ensure that SYSTEM_EMAIL_ADDRESS and ADMIN_EMAIL_ADDRESS are both valid and
```

Comment:
Use `//` for comments instead of `/*`. Also changes made here should be replicated in ` core/tests/build_sources/assets/constants.js`.